src/test/unit/views/single-justice-procedure-press.test.ts

```diff
@@ -101,6 +101,56 @@ describe('Single Justice Procedure List page', () => {
             expect(summary[0].innerHTML).contains(summaryHeadingText, 'Could not find the display summary heading');
         });
 
+        it('should display summary heading', () => {
+            const summary = htmlRes.getElementsByClassName(summaryHeading);
+            expect(summary[1].innerHTML).contains(
+                'Important information',
+                'Could not find the display summary heading'
+            );
+        });
+
+        it('should display important info message', () => {
+            const detail = htmlRes
+                .getElementsByClassName('govuk-details')
+                .item(1)
+                .getElementsByClassName('govuk-details__text');
+            expect(detail[0].innerHTML).contains(
+                'In accordance with the media protocol, additional documents ' +
+                    'from these cases are available to the members of the media on request. ' +
+                    'The link below takes you to the full protocol and further information ' +
+                    'in relation to what documentation can be obtained',
+                'Could not find the display summary heading'
+            );
+        });
+
+        it('should display important info link', () => {
+            const link = htmlRes
+                .getElementsByClassName('govuk-details')
+                .item(1)
+                .getElementsByClassName('govuk-details__text')
+                .item(0)
+                ?.getElementsByTagName('a');
+            expect(link[0].getAttribute('href')).contains(
+                'https://www.gov.uk/government/publications/' +
+                    'guidance-to-staff-on-supporting-media-access-to-courts-and-tribunals/' +
+                    'protocol-on-sharing-court-lists-registers-and-documents-with-the-media-accessible-version',
+                'Could not find the display summary heading'
+            );
+        });
+
+        it('should display important info link name', () => {
+            const link = htmlRes
+                .getElementsByClassName('govuk-details')
+                .item(1)
+                .getElementsByClassName('govuk-details__text')
+                .item(0)
+                ?.getElementsByTagName('a');
+            expect(link[0].innerHTML).contains(
+                'Protocol on sharing court lists, registers ' + 'and documents with the media',
```

Comment:
Hhhmm the expected result can be just one string rather than concatenation of strings

Reply:
Good spot. Updated